index.html

```diff
@@ -0,0 +1,144 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
```

Comment:
add a `/` to close all of this self-closing tags like ` <meta charset="utf-8" /> `

Reply:
pushed the changes